src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -177,52 +275,124 @@ public boolean isMP3() {
+	 * @return True if the audio codec is RealAudio Lossless.
+	 */
+	public boolean isRALF() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.REALAUDIO_LOSSLESS);
+	}
+
+	/**
+	 * @return True if the audio codec is Shorten.
+	 */
+	public boolean isShorten() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.SHORTEN);
+	}
+
+	/**
+	 * @return True if the audio codec is Vorbis.
 	 */
 	public boolean isVorbis() {
 		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.VORBIS);
 	}
 
 	/**
+	 * @return True if the audio codec is TrueHD.
+	 */
+	public boolean isTrueHD() {
+		return getCodecA() != null && (getCodecA().equals(FormatConfiguration.TRUEHD));
+	}
+
+	/**
+	 * @return True if the audio codec is TTA.
+	 */
+	public boolean isTTA() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.TTA);
+	}
+
+	/**
+	 * @return True if the audio codec is WavPack.
+	 */
+	public boolean isWavPack() {
+		return getCodecA() != null && getCodecA().equalsIgnoreCase(FormatConfiguration.WAVPACK);
+	}
+
+	/**
 	 * @return True if the audio codec is WMA.
 	 */
 	public boolean isWMA() {
-		return getCodecA() != null && getCodecA().startsWith("wm");
+		return getCodecA() != null && getCodecA().equals(FormatConfiguration.WMA);
 	}
 
 	/**
-	 * @return True if the audio codec is MPEG-2.
+	 * @return True if the audio codec is WMA Lossless.
 	 */
-	public boolean isMpegAudio() {
-		return getCodecA() != null && getCodecA().equalsIgnoreCase("mp2");
+	public boolean isWMALossless() {
+		return getCodecA() != null && getCodecA().equals(FormatConfiguration.WMALOSSLESS);
 	}
 
 	/**
-	 * @return True if the audio codec is PCM.
+	 * @return True if the audio codec is WMA Pro.
 	 */
-	public boolean isPCM() {
-		return getCodecA() != null && (getCodecA().startsWith("pcm") || getCodecA().equals("LPCM"));
+	public boolean isWMAPro() {
+		return getCodecA() != null && getCodecA().equals(FormatConfiguration.WMAPRO);
 	}
 
 	/**
-	 * @return True if the audio codec is FLAC.
+	 * @return True if the audio codec is WMA Voice.
 	 */
-	public boolean isFLAC() {
-		return getCodecA() != null && getCodecA().startsWith("fla");
+	public boolean isWMAVoice() {
+		return getCodecA() != null && getCodecA().equals(FormatConfiguration.WMAVOICE);
+	}
+
+	/**
+	 * @return True if the audio codec is AC-3, DTS or TrueHD.
+	 */
+	public boolean isNonPCMEncodedAudio() {
+		return isAC3() || isDTS() || isTrueHD() || isDTSHD();
 	}
 
 	/**
 	 * @return True if the audio codec is lossless.
 	 */
 	public boolean isLossless() {
-		return getCodecA() != null && (isPCM() || getCodecA().startsWith("fla") || getCodecA().equals("mlp") || getCodecA().equals("wv") || getCodecA().equals("alac"));
+		return
+			getCodecA() != null && (isPCM() || isFLAC() || isMLP() || isShorten() || isTrueHD() ||
+				isWavPack() || isALAC() || isMonkeysAudio()) || isRALF() || isTTA() || isWMALossless();
```

Comment:
@Nadahar As you saw in my PR, i didn't, because it's uncompressed as well.
AIFC it's a false compressed, Apple used this trick for his new AIFF little-endian